shell/browser/api/electron_api_base_window.cc

```diff
@@ -267,6 +267,7 @@ void BaseWindow::OnWindowWillResize(const gfx::Rect& new_bounds,
 }
 
 void BaseWindow::OnWindowResize() {
+  window_->SaveWindowState();
```

Comment:
Question: can this lead to redundant writes if the window is rapidly resized/ moved? applies to the onwindowmove event as well.

For example, if I am playing with the window position/size and spam changes, will this affect performance?

Reply:
What if we added a debounce? ie. max once per second